apps/web/pages/api/auth/signup.ts

```diff
@@ -192,21 +211,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         return;
       }
     }
-    await prisma.user.upsert({
-      where: { email: userEmail },
-      update: {
-        username,
-        password: hashedPassword,
-        emailVerified: new Date(Date.now()),
-        identityProvider: IdentityProvider.CAL,
-      },
-      create: {
-        username,
-        email: userEmail,
-        password: hashedPassword,
-        identityProvider: IdentityProvider.CAL,
-      },
-    });
+    await upsertUser({ userEmail, hashedPassword, username, orgId: undefined });
```

Comment:
Making an artificial upsert to find/create/update a user as the `prisma.user.upsert` now requires unique where conditions having to pass an orgId as a hard requirement and we don't have it for every use case.